specs/DesignDiscussions/InputLabels.md

```diff
@@ -91,6 +129,42 @@ If we want to allow the host to control formatting of the labels, either in gene
 ```
 Another option is to not put this in host config, and instead control this styling via support of native styling on required and optional labels (i.e. CSS, Xaml Styles, etc). This decision speaks to a larger discussion of the direction of host config that is beyond the scope of this document.
 
+> ### Host side styling
+> 
+> Following the previously mentioned text formatting the proposed set of properties we could allow the host application to override would be almost the same set of properties we provide for configuring a TextBlock: 
+> 
+> | Property | Default Value |HostConfig | Native |
+> | --- | --- | --- | --- |
+> | spacing | "default" | X | |
+> | fontType | "default" | X | X |
+> | size | "default" | X | X |
+> | weight | "default" | X | X |
+> | color | "default" | X | X |
+> | isSubtle | false | X | |
+> | separator | false | X | |
```

Comment:
we may want to remove:
- fontType
- weight (we're supporting markdown)
- isSubtle
- separator
- horizontalAlignment